packages/big-design/src/components/Datepicker/spec.tsx

```diff
@@ -60,39 +60,33 @@ test('renders select label', () => {
   expect(getByText('test')).toBeInTheDocument();
 });
 
-test('calls onDateChange function when a date cell is clicked', () => {
+test('calls onDateChange function when a date cell is clicked', async () => {
   const changeFunction = jest.fn();
   const { container } = render(<Datepicker onDateChange={changeFunction} />);
 
   const input = container.querySelector('input');
 
-  fireEvent.click(input as HTMLInputElement);
+  await userEvent.click(input as HTMLInputElement);
 
   const datepicker = container.querySelector('.react-datepicker');
 
   const cell = datepicker?.querySelector('.react-datepicker__day--today');
 
-  act(() => {
-    fireEvent.click(cell as HTMLElement);
-  });
+  await userEvent.click(cell as HTMLElement);
 
   expect(changeFunction).toHaveBeenCalled();
 });
 
-test('no error when input date value manually', () => {
+test('no error when input date value manually', async () => {
   const changeFunction = jest.fn();
   const { container } = render(<Datepicker onDateChange={changeFunction} />);
 
   const dateString = 'Wed, 03 June, 2020';
   const input = container.querySelector('input');
 
-  fireEvent.input(input as HTMLInputElement, {
-    target: {
-      value: dateString,
-    },
-  });
+  await userEvent.type(input as HTMLInputElement, dateString);
 
-  expect(changeFunction).not.toBeCalled();
+  expect(changeFunction).toBeCalled();
```

Comment:
This is a big issue. Our `Datepicker` component doesn't update date when entered manually in my local environment, but on tests it looks like it does. Will open a github issue for this.

Reply:
#852